HIP-Basic/gpu_arch/gpu_arch_vs2019.sln

```diff
@@ -1,4 +1,4 @@
-﻿
+
 Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Sln's header should start with `Microsoft Visual Studio Solution File, Format Version XX.YY`